kivy/tests/test_window_base.py

```diff
@@ -18,3 +19,37 @@ def test_to_normalized_pos(self):
                 assert result_sy == expected_sy
         finally:
             win.system_size = old_system_size
+
+
+class WindowOpacityTest(GraphicUnitTest):
+
+    def setUp(self):
+        super().setUp()
+        self._prev_window_opacity = self.Window.opacity
+        self._prev_history = LoggerHistory.history[:]
+
+    def tearDown(self):
+        self.Window.opacity = self._prev_window_opacity
+        LoggerHistory.history[:] = self._prev_history
+        super().tearDown()
+
+    def check_opacity_support(self):
+        LoggerHistory.clear_history()
+        self.Window.opacity = 0.2
```

Comment:
Won't work for the case when the current opacity value is set to `0.2`. You must ensure that you set the different value from the current one. The same goes for `test_window_opacity_property` method.

Reply:
@mak8kammerer Can add changes for the case above? With those changes in-place this pull request is pretty much done.

Reply:
Done. Please check and let me know.